float32 deg_angle
---
bool move_successful
---